### DISCUSSION TITLE
Unsuccessful Installation Validation version 11

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#1270] [Questions & Problems]
Dear all,

I wanted to test the OSP version 11 installation on a Windows system installed on a virtual machine, and to check if the software worked correctly I lunched the installation validator. I already performed the same test with previous version on similar machines and it was always successful. 
In this case, for version 11, I get one the the test that fails. I repeated several times the test, reboot the virtual machine, reboot the host machine and performed all the windows updates.
Below I have included the screenshot of the failed test and the simulation that fails the criteria.
Do you have any similar experience? Any idea where this could come from?

Thanks a lot in advance for your help!

Best,
Donato

![image](https://user-images.githubusercontent.com/11489621/182576380-388c138e-68e4-474b-ae23-ad19ff034630.png)

**Comment by Yuri05** [comment]
Can you attach the pdf report and the generated output file 
**Human_MultiORAL_6_12_12_Dissolved-Human_MultiORAL_6_12_12_Dissolved_solubility.json** from your **output** folder?
![grafik](https://user-images.githubusercontent.com/25061876/182597729-98a82660-3622-4be7-9b21-eb7ffaf35746.png)

**Reply by teutonicod** [reply]
Sure here it is! Thanks a lot for the help!

[Open Systems Pharmacology Suite - 11-Installation Validation_08_03_22_7_40_46.pdf](https://github.com/Open-Systems-Pharmacology/Forum/files/9251267/Open.Systems.Pharmacology.Suite.-.11-Installation.Validation_08_03_22_7_40_46.pdf)

[Human_MultiORAL_6_12_12_Dissolved-Human_MultiORAL_6_12_12_Dissolved_solubility.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/9251277/Human_MultiORAL_6_12_12_Dissolved-Human_MultiORAL_6_12_12_Dissolved_solubility.zip)

**Reply by Yuri05** [reply]
Your simulation is valid. If you look at the "problematic" curves (old and new) you will see that the values are in the range `[-9E-15 .. 1E-15]` - which is just a numerical noise (numerical zero) in both cases.
Our comparison algorithm tries to identify the range of "numerical noise" dynamically for every calculated curve - and in this case it's just way too low (`6.25E-17` - you can see it in the generated output).
![grafik](https://user-images.githubusercontent.com/25061876/182621636-622dd89b-a544-4626-9a77-fa83dbc078ba.png)
(If you are interested why something like this happens, check here: https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/issues/1633)

We will try to improve the comparison algorithm in the future and make it not "too sensitive where not required".

So in short: **your installation is valid** 🙂

**Reply by teutonicod** [reply]
OK, thanks a lot! Just to understand, does this happen only on my computer or everybody that lunch the Installation Validator with version 11 find the same result?

**Reply by Yuri05** [reply]
Not everybody for sure (me not, for example :)) - but some people will get this as well I guess
